### PR TITLE
Add support for single and double quotation marks

### LINKED ISSUE #11: Why only Backtick (`). Add support for single & double quotation marks (', ") also.
Generally we use single or double quotation marks to highlight a phrase.

### PR DESCRIPTION
**Pull request by Hanzei**
Fixes #11

### CODE REVIEW

**Comment by Hanzei** [comment]
Escaping a character doesn't work at the moment. Do we want this?

**Comment by kaakaa** [comment]
Thank you for a lot of supports, @Hanzei!

But I feel a little strangeness to this.
With this fix, poll server interpret the follows as correct.
```
/poll `How are you?" :+thumbsup: :thumbsdown:
(start quote is `, end quote is ")
```

If handling this correctly, we must write complicated regular expressions.
It's hard for me.

As an alternative plan
* add quotation marks settings to the `conf.json`. 
  * e.g. `"quotation": "'"`
* change overall format to eliminate the need for quotation marks. 
  * e.g. `/poll :+thumbsup: :thumbsdown:  How are you?`

What do you think?

**Comment by Hanzei** [comment]
> But I feel a little strangeness to this.
> With this fix, poll server interpret the follows as correct.
> ```
> /poll `How are you?" :+thumbsup: :thumbsdown:
> (start quote is `, end quote is ")
> ```
> If handling this correctly, we must write complicated regular expressions.
> It's hard for me.

For me this behaviour is fine for me. But if you want, I can change it. The regular expression isn't so complicated.

> As an alternative plan
> - add quotation marks settings to the conf.json.
>   - e.g. `"quotation": "'"`

I don't want to add an option for this. It seems to complicated.

> - change overall format to eliminate the need for quotation marks.
>    - e.g. `/poll :+thumbsup: :thumbsdown: How are you?`

We can think about this. Reversing the order is not intuitive. What do you think about banning emojis from the question and cut the question at the first emoji?
e.g.: `/poll How are you? :thumbsup: :thumbsdown:`

**Comment by kaakaa** [comment]
I think that it would be better to be enable to use emojis in the question.
So could you please fix the regexp not to accept the question with different first and last quotes?

File: poll/poll_request.go [hunk 1]
```diff
@@ -32,7 +32,17 @@ func NewPollRequest(s string) (*PollRequest, error) {
 				return nil, fmt.Errorf("Unexpected Error: ChannelID in request is empty.")
 			}
 		case "text":
-			re := regexp.MustCompile("`([^`]+)`(.+)")
+      var re *(regexp.Regexp)
```
Comment (kaakaa) [inline poll/poll_request.go]: Our codes have indented by tabs, but your code use two spaces as indent.

File: poll/poll_request_test.go [hunk 2]
```diff
@@ -15,7 +15,17 @@ func TestNewPollRequest(t *testing.T) {
 		err       error
 	}{
 		{"teamidxxxx", "channelidxxxx", "`description` :emoji1:", "description", []string{"emoji1"}, nil},
-		{"teamidxxxx", "channelidxxxx", "`description including space` :emoji1: :emoji2: :emoji3:", "description including space", []string{"emoji1", "emoji2", "emoji3"}, nil},
+    {"teamidxxxx", "channelidxxxx", "'description' :emoji1:", "description", []string{"emoji1"}, nil},
```
Comment (kaakaa) [inline poll/poll_request_test.go]: This has problems about indent.

File: poll/poll_request.go (hunk 1 above)
Comment (kaakaa) [inline poll/poll_request.go]: And I think that it should create new private function for parsing `text` field.

**Pull request merged**